animated_toolbox/css/scss/_layout.scss

```
@@ -0,0 +1,13 @@
+html {
+  scroll-behavior: smooth;
+  font-size: 100%;
+}
+
+body {
+  margin: 0;
+  background-color: $secondaryColLight;
+}
+
+* {
+  box-sizing: border-box;
+}
```

@praxoc This isn't the recipe I linked to in my previous feedback. Please read [Paul Irish's 2012 FTW post](https://www.paulirish.com/2012/box-sizing-border-box-ftw/) and use his code. This guy knows what he's talking about (front-end developer, works on the Google Chrome team.)
```suggestion

/* apply a natural box layout model to all elements, but allowing components to change */
html {
  box-sizing: border-box;
}
*, *:before, *:after {
  box-sizing: inherit;
}

```

---

Awesome. Will modify the code now. Thanks !